### DISCUSSION TITLE
Where can I find the documentation about the API between client and server

### DISCUSSION THREAD

**Discussion by richardbrinkman** [JiHong88/SunEditor#1177] [Q&A] [answered]
I would like to use SunEditor to upload and retrieve images to a PHP backend. Therefore, I need a description about the API that is being used between the browser and the server. Specifically, I need the format of the JSON responses my server backend should send back to SunEditor.

From the example https://etyswjpn79.execute-api.ap-northeast-1.amazonaws.com/suneditor-demo (found on http://suneditor.com/sample/html/options.html) I extracted the response for the imageGallery as:

```javascript
{
    responseCode: 200,
    result: [
        {
            src: "https://...",
            name: "...",
            alt: "...",
            tag: "..."
        }
    ]
}
```
I am looking for a similar format description my upload backend should return. I tried the same format as the imagegallery, but that did not work.

**Comment by JiHong88** [comment, marked as answer]
Search for "imageUploadUrl" here https://github.com/JiHong88/suneditor#options.
